front/src/components/SocialNetwork/SocialNetwork.js

```diff
@@ -110,6 +111,22 @@ export const SocialNetwork = () => {
     }
   };
 
+  const removeFriend = (user) => {
```

Comment:
@SidhaantAnand Take a look here to see why we are getting a CORS policy error (as this is our first POST request)
<img width="1670" alt="Screen Shot 2020-12-02 at 3 19 52 AM" src="https://user-images.githubusercontent.com/35234448/100854047-7c2ba900-3456-11eb-86f6-659065e7987a.png">

Reply:
Fixed.